libcloud/container/drivers/lxd.py

```diff
@@ -971,6 +1073,25 @@ def list_images(self):
             images.append(self.ex_get_image(fingerprint=fingerprint))
         return images
 
+    def ex_has_image(self, alias):
+        """
+        Helper function. Returns true is the image with the
+        given alias exits on the host
+
+        :param alias: the image alias
+        :type  alias: ``str``
+
+        :rtype: boolean
```

Comment:
Looks like method actually returns a tuple, please update the comment accordingly.